+euphonic/install_python_modules.m

```diff
@@ -6,26 +6,34 @@ function install_python_modules()
     % >>  euphonic.install_python_modules
 
     req_mods = required_modules;
-    try
-        verify_python_modules(req_mods{:});
-    catch ME
-        % Checks we have the right Python version (>=3.7)
-        if strcmp(ME.identifier, 'euphonic_horace_interface:verify_python_modules:pythonVersion')
-            rethrow(ME);
-        end
-    end
+
     pipe = py.subprocess.PIPE;
     kwargs = pyargs('stdout', pipe, 'stderr', pipe);
     mods_kw = {};
     for ii = 1:2:numel(req_mods)
-        mods_kw = [mods_kw {[req_mods{ii} '==' req_mods{ii+1}]}];
+        mods_kw = [mods_kw req_mods{ii}];
     end
-    out = py.subprocess.run([{py.sys.executable '-m' 'pip' 'install'} mods_kw], kwargs);
+    % Get Python execuable
+    try
+        % Pyenv only introduced in 2019b
+        py_exe = pyenv().Executable;
+    catch ME
+        if (strcmp(ME.identifier,'MATLAB:UndefinedFunction'))
+            % Pyversion is deprecated but use if pyenv not available
+            [ver, py_exe, isloaded] = pyversion(py_ex_path);
```

Comment:
I think this should be: `[~, py_exe, ~] = pyversion` (we haven't defined a `py_ex_path` and don't need the other variables.)